src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -513,6 +519,33 @@ int32_t CryptoNative_SetCiphers(SSL_CTX* ctx, const char* cipherList, const char
     return ret;
 }
 
+int32_t CryptoNative_SetCiphers(SSL* ssl, const char* cipherList, const char* cipherSuites)
+{
+    int32_t ret = true;
+
+    // for < TLS 1.3
+    if (cipherList != NULL)
+    {
+        ret &= SSL_set_cipher_list(ssl, cipherList);
+        if (!ret)
+        {
+            return ret;
+        }
+    }
+
+    // for TLS 1.3
+#if HAVE_OPENSSL_SET_CIPHERSUITES
+    if (CryptoNative_Tls13Supported() && cipherSuites != NULL)
+    {
+        ret &= SSL_set_ciphersuites(ssl, cipherSuites);
+    }
```

Comment:
I'm not following this logic.  If TLS 1.3 is supported and cipherSuites != null, we call SSL_set_ciphersuites twice?

Reply:
Nevermind, it's two different methods.  For TLS 1.3 we need to call both SSL_set_cipher_list and SSL_set_ciphersuites?

Reply:
Nevermind.  Two different inputs, two different methods.  I need more coffee.